_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -26,101 +26,34 @@ To enable searchable snapshots, reference the [instructions](https://opensearch.
 
 ## Using a snapshot for search
 
-OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. You can configure the amount of storage available for caching snapshot data or use the system defaults. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
+OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
 
-For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy, removing the least recently used segments first.
+For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy.
 
 ## Performance
 
-When cached data is searched repeatedly, searchable snapshots provide consistent search performance and throughput. Performance is slow the first time the snapshot data is accessed, but subsequent searches are comparable in performance to regular OpenSearch cluster searches. Note, however, that the ratio of cache size to the combined size of the searchable snapshot, along with the search pattern, will factor into performance. The more data the cache misses, the lower the overall performance. In the next section, we cover the performance results of our benchmark testing.
+Comparing the performance of searchable snapshot queries and local index queries on the StackOverflow dataset, the initial run of Searchable Snapshots took approximately twice as long as local index. However, subsequent queries showed similar performance for both searchable snapshots and local index because searchable snapshots used cached data.
```

Comment:
"took approximately twice as long" needs to be substantiated - this is the right data to do so, we should probably add to the benchmarking section: https://quip-amazon.com/5rEsA5Jg2ugP/Searchable-Snapshots-Performance-Testing#temp:C:CWL8e4748a55b854c22af4cfef1b

Reply:
Agreed, lets add the section from the quip noted above. (Scan time)